pkgs/build-support/dotnet/dotnetenv/build-solution.nix

```diff
@@ -31,41 +31,37 @@ stdenv.mkDerivation {
   '';
 
   installPhase = ''
-    addDeps()
-    {
-	if [ -f $1/nix-support/dotnet-assemblies ]
-	then
-	    for i in $(cat $1/nix-support/dotnet-assemblies)
-	    do
-		windowsPath=$(cygpath --windows $i)
-		assemblySearchPaths="$assemblySearchPaths;$windowsPath"
-
-		addDeps $i
-	    done
-	fi
+    runHook preInstall
```

Comment:
IMO this functional change should be in a separate commit.

Reply:
It's a simple change so I simply added it to the commit msg.

Reply:
Made it one line